src/components/notebook-utils.js

```diff
@@ -91,7 +91,7 @@ export const toolExtensionDisplay = {
 }
 
 export const getPatternFromTool = toolLabel => Utils.switchCase(toolLabel,
-  [tools.RStudio.label, () => '.*\\.R(md)'],
+  [tools.RStudio.label, () => '.+(\\.R|\\.Rmd)$'],
```

Comment:
Does the above work for you? I don't think it would since what are escaped are the slashes (vs. periods).

```suggestion
  [tools.RStudio.label, () => '.+(\.R|\.Rmd)$'],
```

Reply:
you need double slash...it's not happy with one slash in VS code...we have double slash for .ipynb as well